src/components/DashboardComponent/DashboardComponent.jsx

```diff
@@ -19,29 +23,49 @@ class DashboardComponent extends Component {
       style: { width: "0" }
     };
   }
+
   componentDidMount() {
     if (this.state.id) {
       //wrapped in a conditional for PWA
       this.props.dispatchLoadUser(this.state.id.id);
       this.props.dispatchGetDiaryData(this.state.date);
+      this.props.dispatchLoadWorkouts(this.state.id.id);
     }
   }
 
   componentDidUpdate(prevProps) {
-    if (this.props.diaryData !== prevProps.diaryData) {
+    if (
+      this.props.diaryData !== prevProps.diaryData ||
+      this.props.workouts !== prevProps.workouts
+    ) {
       let goal = this.props.users && this.props.users.recommended_calories;
+
       let food =
         this.props.diaryData &&
         this.props.diaryData.reduce((total, data) => {
           return total + data.calories;
         }, 0);
-      if (Math.sign(goal - food) === -1) {
+
+      //gets calories burned by today's date
+      let exercise =
+        this.props.workouts &&
+        this.props.workouts
+          .filter(data => {
+            let todaysDate = moment(this.state.date).format("YYYY-MM-DD");
+            let getCreatedDate = moment(data.created_at).format("YYYY-MM-DD");
```

Comment:
just a side note, this needs to be changed when Janelle's branch gets pushed up.